src/db/add_package.rs

```diff
@@ -130,6 +130,29 @@ pub(crate) fn add_package_into_database(
     Ok(release_id)
 }
 
+pub(crate) fn add_doc_coverage(
+    conn: &mut Client,
+    release_id: i32,
+    doc_coverage: DocCoverage,
+) -> Result<i32> {
+    debug!("Adding doc coverage into database");
+    let rows = conn.query(
+        "INSERT INTO doc_coverage (release_id, total_items, documented_items)
```

Comment:
Why is this a separate table?

Reply:
We plan to add extra data. `doc-coverage` will be extended into the future and @jyn514 also wanted to include the current data (the by-module results). Since we didn't agree on how to display the information, we decided to do it in a later PR.

Reply:
Hmm ok.